### PR TITLE
Add CData Software

### PR DESCRIPTION
**Pull request by JerodJ**

### CODE REVIEW

**Comment by rosa** [comment]
Hi @JerodJ! First of all, sorry for the delay, we haven't been paying enough attention to these pull requests lately. I just had a look at the CData integrations page but it looks like you don't offer an app that integrates with Basecamp, but [rather a collection of wrappers of the API](https://www.cdata.com/drivers/basecamp/docs/) (named _drivers_)? Maybe the category number 9 (_Developer-only Tools_) would be more appropriate.

I have also another question: are these wrappers for the Basecamp 3 API? I haven't seen any reference to a Basecamp version, and the Basecamp logo used in the page is certainly Basecamp 2's logo.

**Comment by JerodJ** [comment]
Hi @rosa! The drivers can be used in a variety of ways. Primarily, they provide access to Basecamp data from popular BI, reporting, and ETL tools (like Tableau, MS Excel, MS Power BI, QlikView, etc) without writing any code, hence my slotting them into _Reporting, Charts, Planning_ section. They can also be used by developers, but certainly aren't _Developer-only Tools_.

The drivers definitely support Basecamp 3. I'll reach out to the web team about the logo.

**Comment by rosa** [comment]
@JerodJ thanks! In that case, maybe category 1 would be a better fit? That's _Mobile and Desktop Apps_, and it's the category that apps such as Zapier or Unito are using. That is, it's not that your integration is itself a _Reporting, Charts, Planning_ app, but that it can be used to connect these kind of apps into Basecamp. What do you think?

**Comment by JerodJ** [comment]
@rosa I've updated the listing use category 1. Thanks for the prompt follow-up!

**Comment by rosa** [comment]
Thank you!

File: integrations.yml [hunk 1]
```diff
@@ -275,10 +275,16 @@
...
+  icon: "cdata.png"
+  category: 1
 -
   name: "Akita"
   description: "Monitor the status of your Basecamp projects in Akita's powerful Customer Success Manage Platform."
   website: "https://www.akitaapp.com?utm_source=basecamp"
   icon: "akita.png"
   category: 7
-
+-
```
Comment (rosa) [inline integrations.yml]: @JerodJ it looks like this change leaked into your legit changes, and broke the Travis build. I imagine it was after fixing conflicts. Would you mind fixing it? I'll merge this after 😄

Comment (JerodJ) [reply integrations.yml]: @rosa fixed

Comment (rosa) [reply integrations.yml]: Perfect, thanks!

**Pull request merged**